<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shutdownWindow</class>
 <widget class="QDialog" name="shutdownWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>174</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>539</width>
    <height>174</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>539</width>
    <height>174</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Noto Serif CJK SC</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>注意</string>
  </property>
  <widget class="QPushButton" name="btCancel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>124</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btReboot">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>124</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>重启</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btShutdown">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>124</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLogo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/icons/icons/reboot.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lbTip">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请开始你的操作</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
